vhdbuilder/packer/components.json

```diff
@@ -73,7 +74,8 @@
       "versions": [
         "v1.1.8",
         "v0.0.7",
-        "v0.0.6"
+        "v0.0.6",
+        "v0.0.6_v0.0.5"
```

Comment:
why do we need this? are we not using the latest version 1.1.8?

Reply:
will remove it in next PR after we remove the values in RP toggle.